### PR TITLE
[patch] feature/internal/cassandra/add option

### PR DESCRIPTION
**Pull request by kpango**
### Description:
Add Cassandra 2 Option
- TokenAwareHostPolicy
- SerialConsistency

### Environment:

- Golang Version: 1.14
- Docker Version: 19.03.5
- Kubernetes Version: 1.17.3
- NGT Version: 1.9.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [x] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: charts/vald/values.yaml [hunk 1]
```diff
@@ -1757,6 +1761,8 @@ meta:
       num_conns: 2
       # meta.cassandra.config.consistency -- consistency type
       consistency:  quorum
+      # backupManager.cassandra.config.searial_consistency -- read consistency type
```
Comment (rinx) [inline charts/vald/values.yaml:1764]:
```suggestion
      # meta.cassandra.config.serial_consistency -- read consistency type
```

File: charts/vald/values.yaml [hunk 2]
```diff
@@ -1838,6 +1844,8 @@ meta:
         non_local_replicas_fallback: false
         # meta.cassandra.config.pool_config.shuffle_replicas -- shuffle replica enabled
         shuffle_replicas: false
+        # backupManager.cassandra.config.pool_config.token_aware_host_policy -- token aware host policy enabled
```
Comment (rinx) [inline charts/vald/values.yaml:1847]:
```suggestion
        # meta.cassandra.config.pool_config.token_aware_host_policy -- token aware host policy enabled
```

File: charts/vald/values.yaml [hunk 3]
```diff
@@ -1255,6 +1255,8 @@ backupManager:
       num_conns: 2
       # backupManager.cassandra.config.consistency -- consistency type
       consistency:  quorum
+      # backupManager.cassandra.config.searial_consistency -- read consistency type
```
Comment (rinx) [inline charts/vald/values.yaml:1258]:
```suggestion
      # backupManager.cassandra.config.serial_consistency -- read consistency type
```

File: charts/vald/values.yaml [hunk 4]
```diff
@@ -1255,6 +1255,8 @@ backupManager:
...
+      searial_consistency:  local_serial
```
Comment (rinx) [inline charts/vald/values.yaml:1259]:
```suggestion
      serial_consistency:  local_serial
```

File: charts/vald/values.yaml [hunk 5]
```diff
@@ -1757,6 +1761,8 @@ meta:
       num_conns: 2
       # meta.cassandra.config.consistency -- consistency type
       consistency:  quorum
+      # backupManager.cassandra.config.searial_consistency -- read consistency type
+      searial_consistency:  local_serial
```
Comment (rinx) [inline charts/vald/values.yaml:1765]:
```suggestion
      serial_consistency:  local_serial
```

File: charts/vald/values.yaml [hunk 6]
```diff
@@ -1336,6 +1338,15 @@ backupManager:
...
         shuffle_replicas: false
+        # backupManager.cassandra.config.pool_config.token_aware_host_policy -- token aware host policy enabled
+        token_aware_host_policy: false
+      host_filter:
+        # meta.cassandra.config.host_filter.enabled -- enables host filtering
+        enabled: false
+        # meta.cassandra.config.host_filter.data_center -- name of data center of filtering target
+        data_center: ""
+        # meta.cassandra.config.host_filter.white_list -- list of white_list which allows each connection
+        white_list: []
```
Comment (rinx) [inline charts/vald/values.yaml:1349]:
```suggestion
      host_filter:
        # backupManager.cassandra.config.host_filter.enabled -- enables host filtering
        enabled: false
        # backupManager.cassandra.config.host_filter.data_center -- name of data center of filtering target
        data_center: ""
        # backupManager.cassandra.config.host_filter.white_list -- list of white_list which allows each connection
        white_list: []
```

**Comment by rinx** [comment]
Others OK :+1:

File: charts/vald/values.yaml (hunk 6 above)
Comment (kpango) [reply charts/vald/values.yaml:1349]: OMG

**Pull request merged**